### PR TITLE
Refactor signup page

### PR DESCRIPTION
**Pull request by Blackfry**

### CODE REVIEW

File: apps/marketplace/components/SignupForm/SignupForm.js [hunk 1]
```diff
@@ -1,371 +1,299 @@
 /* eslint-disable camelcase */
 import React from 'react'
 import PropTypes from 'prop-types'
-import { connect } from 'react-redux'
 import { Form } from 'react-redux-form'
 import PageAlert from '@gov.au/page-alerts'
 
 import { required, validEmail, governmentEmail } from '../../components/validators'
 import Layout from '../../components/shared/Layout'
```
Comment (matt-sm) [inline apps/marketplace/components/SignupForm/SignupForm.js]: we shouldn't put all the extra blank lines in the imports:
- absolute
- blank line
- relative
- blank line
- css modules

File: apps/marketplace/components/SignupForm/SignupForm.js [hunk 2]
```diff
@@ -1,371 +1,299 @@
...
-  signupSuccess: state.user.signupSuccess,
-  currentlySending: state.app.currentlySending
-})
-
-const mapDispatchToProps = dispatch => ({
-  handleSignupSubmit: model => dispatch(handleSignupSubmit(model))
-})
+SignupForm.defaultProps = {
+  csrf_token: '',
+  signupSuccess: null
```
Comment (matt-sm) [inline apps/marketplace/components/SignupForm/SignupForm.js]: should that be `false`

File: apps/marketplace/components/SignupForm/SignupForm.js [hunk 3]
```diff
@@ -1,371 +1,299 @@
...
+            </Layout>}
         </div>
       </div>
-    )
-  }
+    </div>
+  )
 }
 
 SignupForm.propTypes = {
```
Comment (matt-sm) [inline apps/marketplace/components/SignupForm/SignupForm.js]: I think your missing `submitClicked`

File: apps/marketplace/components/SignupForm/SignupForm.js (hunk 2 above)
Comment (Blackfry) [reply apps/marketplace/components/SignupForm/SignupForm.js]: I normally use null for defaults so if it is false I know it's been set that way from an action

**Pull request merged**